app/pages/capability-selector/capabilitySelectorPageContext.test.js

```diff
@@ -0,0 +1,134 @@
+import { createCapabilitySelectorPageContext } from './capabilitySelectorPageContext';
+import manifest from './manifest.json';
+
+describe('createCapabilitySelectorPageContext', () => {
+  it('should create a context for the capability selector page', () => {
```

Comment:
Don't think we need this test. All your other test are more specific and cover all this.

Reply:
are you suggesting we delete a test?? 😱

Reply:
I know. Shock right!!! 🙀

Reply:
the whole file?